views/block.tmpl

```diff
@@ -26,7 +26,7 @@
                     {{if .NextHash}}
                     Next Block Hash: <span style="float: right;font: 12px monospace;" >{{template "copyButton" .NextHash}}<a href="/explorer/block/{{.NextHash}}">{{.NextHash}}</a></span><br>
                     {{end}}
-                    Transactions: <span style="float: right;" >{{.TxCount}}</span><br>
+                    Transactions: <span style="float: right;" >{{.RawTx | len}}</span><br>
```

Comment:
do we need to pipeline this here?  Maybe I'm forgetting how this works, but I though you could just do `{{len .RawTx}}`

Reply:
Forgot I could do that, so used to working with the mapped functions